examples/with-dynamic-import/pages/index.js

```diff
@@ -41,6 +44,23 @@ const IndexPage = () => {
       {/* Load on demand */}
       {showMore && <DynamicComponent5 />}
       <button onClick={() => setShowMore(!showMore)}>Toggle Show More</button>
+
+      {/* Load library on demand */}
+      <div style={{ marginTop: '1rem' }}>
+        <input
+          type="text"
+          placeholder="Search"
+          onChange={async (e) => {
+            const { value } = e.currentTarget
+            // Dynamically load fuse.js
+            const Fuse = (await import('fuse.js')).default
+            const fuse = new Fuse(names)
```

Comment:
Should we memoize the result so that these lines would only need to be run once?

Reply:
@chibicode I think that's not needed for the sake of the demo, it should be very fast in any case. This action doesn't delay user input and the module will only be imported once.